docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,143 @@
+├── quickstart_etl_tests
+├── dagster_cloud.yaml
+├── pyproject.toml
+├── setup.cfg
+└── setup.py
+```
+
+## Code Locations
+
+The only **mandatory** section in the dagster_cloud.yaml file is the locations key that specifies a list of code locations that should be deployed to Dagster Cloud. Each item in the list of locations must also include a location_name key that specifies the name of a location and a code_source key that specifies where to find the definitions in the code location.
+Parameter reference table:
+
+| Parameter                            | Description                                                                           | Data type            |
+| ------------------------------------ | ------------------------------------------------------------------------------------- | -------------------- |
+| `locations.location_name`            | The name of your code location that will appear in the Dagster UI code locations page | string               |
+| `locations.code_source.package_name` | Use if your dagster code is structured as a package                                   | string (folder name) |
+| `locations.code_source.python_file`  | Use if your dagster code is in a single file                                          | string (.py file)    |
+| `locations.code_source.module_name`  | The module that contains your dagster code (which matches pyproject.toml)                                            | string (module name) |
+
+Example for a single code location:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+```
+
+Example for multiple code locations:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+
+## Configuring your working directory
+
+There might be some cases where you want to run your Dagster code from a different directory than the root of your code repository. You can specify the directory you want to run your Dagster code from in the dagster_cloud.yaml file using the `working_directory` option.
+
+Parameter reference table:
+
+| Parameter           | Description                                            | Data type     |
+| ------------------- | ------------------------------------------------------ | ------------- |
+| `working_directory` | The path of the directory that Dagster should run from | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+## Build directory
+
+Setting a build directory is useful if your setup.py or requirements.txt is not at your project root but in a subdirectory. This is common if you have multiple python packages within a single dagster project.
```

Comment:
```suggestion
Setting a build directory is useful if your `setup.py` or `requirements.txt` is in a subdirectory instead of the project root. This is common if you have multiple Python packages within a single Dagster project.
```

Formatting and capitalization.